app/src/NativeBase/Components/Project/ProjectAttachments.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file displays the project attachment link
+ */
+import React from 'react'
+import { navigate } from '@/Navigators/utils'
+import { Link } from 'native-base'
+
+interface ProjectAttachmentsProps {
+  details: string
+  url: string
+}
+
+/**
+ * Displays the attachment link
+ * @param {ProjectAttachmentProps} props the props for this component
+ * @param {string} props.details details the text for the attachment button
+ * @param {string} props.url url of attachment, the project scope pdf
+ * @returns {ReactElement|null} renders the attachment link, or null if not valid
+ */
+const ProjectAttachments = ({ details, url }: ProjectAttachmentsProps) => {
+  // Project Attachments will not be displayed if URL is null or not a validate format
+  if (!url || !url.match('drive.google.com')) return null
+  return (
+    <Link
+      alignSelf="flex-start"
```

Comment:
Do we need the `alignSelf` and `margin` properties here?  I tried it without (added a random file to Made 4 U in ML2 project) and it looked the same either way

Reply:
Yes oddly it does look the same (I recall it being centred), have removed it.